docker/Dockerfile

```diff
@@ -17,6 +17,9 @@
 
 FROM google/cloud-sdk
 
+ARG IMAGE_TAG
+ENV image_tag=${IMAGE_TAG}
```

Comment:
I think, given that it's the COMMIT_SHA we should call it that.  Otherwise users that run

  `docker run gcr.io/gcp-variant-transforms/gcp-variant-transforms:1.0 echo $image_tag`

will see a commit SHA1 instead of the tag they actually pulled...